about_binaries.exs

```diff
@@ -31,4 +31,65 @@ defmodule AboutBinaries do
     contatenated = "Hello " <> "world"
     assert contatenated == __?
   end
+
+  @tag :focus
+  think "Numbers can be represented as bits" do
+    bit_representation = fn (integer) -> Integer.to_string(integer, 2) end
```

Comment:
I like that the variable name helps communicate the intention of this code. I was going to say, we might even go for more explicitness with the description of the exercise. Something like "Base-2 (binary) numbers are represented as bits"